scripts/deploy_gke.sh

```diff
@@ -64,23 +64,36 @@ if [[ $ENV == "autopush" ]]; then
     echo $(gsutil cat "$src") >> deploy/storage/bigtable_import_groups.version
   done
 fi
-export PROJECT_ID=$(yq eval '.project' deploy/gke/$ENV.yaml)
-export REGION=$(yq eval '.region' deploy/gke/$ENV.yaml)
-export IP=$(yq eval '.ip' deploy/gke/$ENV.yaml)
-export DOMAIN=$(yq eval '.domain' deploy/gke/$ENV.yaml)
-export API_TITLE=$(yq eval '.api_title' deploy/gke/$ENV.yaml)
-export API=$(yq eval '.api' deploy/gke/$ENV.yaml)
+export PROJECT_ID=$(yq eval '.mixer.gcpProjectID' deploy/helm_charts/envs/$ENV.yaml)
+export REGION=$(yq eval '.region' deploy/helm_charts/envs/$ENV.yaml)
+export IP=$(yq eval '.ip' deploy/helm_charts/envs/$ENV.yaml)
+export DOMAIN=$(yq eval '.mixer.serviceName' deploy/helm_charts/envs/$ENV.yaml)
+export API_TITLE=$(yq eval '.api_title' deploy/helm_charts/envs/$ENV.yaml)
+export API=$(yq eval '.api' deploy/helm_charts/envs/$ENV.yaml)
 export CLUSTER_NAME=mixer-$REGION
 
-cd $ROOT/deploy/overlays/$ENV
-
 # Deploy to GKE
-kustomize edit set image gcr.io/datcom-ci/datacommons-mixer=gcr.io/datcom-ci/datacommons-mixer:$TAG
-kustomize build > kustomize-build.yaml
-cp kustomization.yaml kustomize-deployed.yaml
 gcloud config set project $PROJECT_ID
 gcloud container clusters get-credentials $CLUSTER_NAME --region $REGION
-kubectl apply -f kustomize-build.yaml
+
+# Change "mixer_prod" for example, to "mixer-prod"
+RELEASE=${ENV//_/-}
+
+# Create a release specific image for the deployment.
+./scripts/push_binary.sh $TAG
```

Comment:
is this needed, at this stage, a version with $TAG should already exist?

Reply:
It didn't when I was running locally. Sometimes I forget to run push_binary.sh beforehand, thought I'd just add it here. But it should only run if image does not exist, added that condition. Thanks!